### PR TITLE
fix issue #7556 turn "Language for 'X' not found" to warnings

### PR DESCRIPTION
**Pull request by ronabet**
Signed-off-by: Ron Nabet <ron.nabet@sap.com>

#### What it does

Fixes https://github.com/eclipse-theia/theia/issues/7556

Converts all errors from the same built-in extension (languages) to warnings.

#### How to test

Start Theia from master.

#### Review checklist

- [x] as an author, I have thoroughly tested my changes and carefully followed [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#requesting-a-review)

#### Reminder for reviewers

- as a reviewer, I agree to behave in accordance with [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#reviewing)

### CODE REVIEW

**Comment by amiramw** [comment]
why is the grouping to array needed?

**Comment by ronabet** [comment]
> why is the grouping to array needed?

In order to connect all the errors to one string (warning).

**Comment by amiramw** [comment]
> In order to connect all the errors to one string (warning).

why do they need to be in one string? can't there be several warnings?

**Comment by ronabet** [comment]
> > In order to connect all the errors to one string (warning).
> 
> why do they need to be in one string? can't there be several warnings?

also a possibility, @akosyakov offered to report them once.

File: packages/plugin-ext/src/main/browser/plugin-contribution-handler.ts [hunk 1]
```diff
@@ -253,6 +254,14 @@ export class PluginContributionHandler {
             }
         }
 
+        if (contributions.viewsWelcome) {
```
Comment (amiramw) [inline packages/plugin-ext/src/main/browser/plugin-contribution-handler.ts:257]: is this related to logging?

**Comment by amiramw** [comment]
> also a possibility, @akosyakov offered to report them once.

I would prefer to just change error to warning and avoid extra complexity

**Comment by ronabet** [comment]
> > also a possibility, @akosyakov offered to report them once.
> 
> I would prefer to just change error to warning and avoid extra complexity

updated

**Comment by amiramw** [comment]
@ronabet please change your commit subject/description to something that explain what it does and not only to have the issue id

**Review by amiramw — APPROVED** [review]
Tested it and "Language for 'X' not found" are now logged as warnings
